wagtail/admin/wagtail_hooks.py

```diff
@@ -212,6 +214,7 @@ def register_workflow_tasks_menu_item():
     return WorkflowTasksMenuItem(
         _("Workflow tasks"),
         reverse("wagtailadmin_workflows:task_index"),
+        name="workflow-tasks",
```

Comment:
There's a few more further down below in this file,

https://github.com/wagtail/wagtail/blob/c5f238f98d5d23ed9f4eb4162128d7de3f2dbc8c/wagtail/admin/wagtail_hooks.py#L956-L973

https://github.com/wagtail/wagtail/blob/c5f238f98d5d23ed9f4eb4162128d7de3f2dbc8c/wagtail/admin/wagtail_hooks.py#L926-L933

Reply:
I'd suggest searching for `MenuItem(` throughout the codebase, as there might be more that I missed.